lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,33 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance
```

Comment:
`aws_sdk_type AWS::RDS::DBInstance, id: db_instance_identifier`

This means that if you create a resource like:

```ruby
instance = rds.db_instances['foo']
aws_rds_instance instance do
  iops 2000
end
```

It should correctly create the resource when the `name` attribute is a `AWS::RDS::DBInstance` object instead of a String.